WPMedia/fusion-news-theme-blocks
# Fix/asub 8199 regwall safari

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

This is a small fix for Safari where the RegWall button is hidden below the toolbar. This only effects Safari and. no other browser.

## Jira Ticket

- [ASUB-8199](https://arcpublishing.atlassian.net/browse/ASUB-8199)

## Acceptance Criteria

Registration Wall should be above the fold on Safari and Chrome

## Test Steps
- Create a page with the Paywall block
- Add the RegWall
- You will need to comment some code and return only the RegWall in the paywall theme block. This will allow you to see the RegWall without any logic to trigger a showing of the RegWall.
```
	return (
		<RegwallOffer
			actionText={registerActionText}
			actionUrl={registerActionUrl}
			displayMode={displayMode}
			headlineText={registerHeaderText}
			linkPrompt={payLinkPrompt}
			linkText={linkText}
			linkUrl={linkUrl}
			reasonPrompt={registerReasonPrompt}
			subheadlineText={registerSubHeaderText}
			className={BLOCK_CLASS_NAME}
		/>
	);
```
- You will need to either open the Paywall page on a iPhone device using the IP address or you can open it in the Xcode simulator. You can only test Chrome on a mobile device.

## Effect Of Changes

### Before
<img width="409" alt="Screenshot 2024-03-14 at 7 28 18 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/5377648/b1ff9f68-d2ee-4913-b0ed-2263d3fa307f">

### After
<img width="409" alt="Screenshot 2024-03-14 at 7 28 18 PM" src="https://github.com/WPMedia/arc-themes-blocks/assets/5377648/abca7051-5d2f-4e6d-af99-c2203c866699">

![IMG_60E0869E7B74-1](https://github.com/WPMedia/arc-themes-blocks/assets/5377648/04edeb3b-84ef-4e78-9725-87cab52d93dd)


## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[ASUB-8199]: https://arcpublishing.atlassian.net/browse/ASUB-8199?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
